packages/dashboard/src/layout/useLayoutManager.ts

```diff
@@ -0,0 +1,18 @@
+import { useContext } from 'react';
+import LayoutManager from '@deephaven/golden-layout';
+import LayoutManagerContext from './LayoutManagerContext';
+
+/**
+ * Retrieve the current LayoutManager from the context
+ */
+function useLayoutManager(): LayoutManager {
+  const layout = useContext(LayoutManagerContext);
```

Comment:
Might as well change this variable name while we're renaming it to `layoutManager`